subprojects/gradle/performance/src/main/kotlin/com/avito/performance/PerformanceExtension.kt

```diff
@@ -1,14 +1,28 @@
 package com.avito.performance
 
-open class PerformanceExtension {
+import org.gradle.api.model.ObjectFactory
+import org.gradle.kotlin.dsl.property
+import javax.inject.Inject
 
-    /* Результаты перформанс-тестов, резултаты сравнения с предыдущим прогоном на девелопе */
-    lateinit var output: String
+open class PerformanceExtension @Inject constructor(objects: ObjectFactory) {
 
-    /* Название файла с результатами теста */
-    lateinit var performanceTestResultName: String
+    /**
+     * Name of a directory for performance test results
+     */
+    val output = objects.property<String>()
```

Comment:
Why don't we use a Directory, FileCollenction type?

Reply:
it would be an API change; i tried to avoid it for now